packages/nextcloud/test/news_test.dart

```diff
@@ -22,15 +22,18 @@ Future run(final DockerImage image) async {
       container = await getDockerContainer(image);
       client = await getTestClient(container);
     });
-    tearDown(() => container.destroy());
+    tearDown(() async {
+      print(await container.collectLogs());
```

Comment:
how chatty is this going to be?
Looks like it currently only outputs linebreaks.

Reply:
Ah forgot to remove it again. It would be perfect if the test package would have a way to execute code when a test fails